buildSystem/dependencies.gradle

```diff
@@ -54,6 +60,12 @@ ext {
             koinandroidxscope         : "org.koin:koin-androidx-scope:${koinandroidxscope}",
             kointest                  : "org.koin:koin-test:${kointest}",
             androidxjunit             : "androidx.test.ext:junit:${androidxjunit}",
-            androidxespressocore      : "androidx.test.espresso:espresso-core:${androidxespressocore}"
+            androidxespressocore      : "androidx.test.espresso:espresso-core:${androidxespressocore}",
+            roomruntime               : "androidx.room:room-runtime:${roomruntime}",
+            roomktx                   : "androidx.room:room-ktx:${roomktx}",
+            roomtest                  : "androidx.room:room-testing:${roomtest}",
```

Comment:
I think you dont need this one

Reply:
good catch, didn't realize i wasn't using it, thanks